packages/runtime-core/src/createRenderer.ts

```diff
@@ -653,7 +653,7 @@ export function createRenderer<
     } else {
       // update content
       const target = (n2.target = n1.target)!
-      if (patchFlag === PatchFlags.TEXT) {
+      if (patchFlag & PatchFlags.TEXT) {
```

Comment:
Not sure about the intention here

Reply:
My bad, this is a legit catch. It's possible for the portal to have a compound flag.